works.yaml

```diff
@@ -1,4 +1,21 @@
 ---
+
+The Lounge:
+  uri:
+  - https://thelounge.github.io
+  description:
+  - The self-hosted web IRC client
```

Comment:
Please make this a complete sentence (preferably with the name of the work in it).